org.eclipse.xtext/src/org/eclipse/xtext/preferences/MapBasedPreferenceValues.java

```diff
@@ -34,7 +38,7 @@ public MapBasedPreferenceValues(Map<String, String> values) {
 	}
 	
 	public MapBasedPreferenceValues() {
-		this(null, Maps.<String, String>newLinkedHashMap());
+		this(null, Maps.newLinkedHashMap());
```

Comment:
Isn't this the same as `new LinkedHashMap<>()`?

Reply:
It is.